apps/www/app/_utils/redirects.server.ts

```diff
@@ -0,0 +1,68 @@
+import { redirect } from 'react-router';
+
+export const designsystemetRedirects = (pathname: string) => {
+  const redirectUrl = redirects.find(
+    (r) =>
+      r.from === pathname.toLowerCase() ||
+      r.from + '/' === pathname.toLowerCase(),
+  );
+  if (redirectUrl) {
+    return redirect(redirectUrl.to);
+  }
+  return null;
+};
+
+const redirects = [
+  {
+    from: '/komponenter',
+    to: '/no/components',
+  },
+  {
+    from: '/bloggen',
+    to: '/no/blog',
+  },
+
+  {
+    from: '/monstre',
+    to: '/no/patterns',
+  },
+  {
+    from: '/grunnleggende',
+    to: '/no/fundamentals',
+  },
+];
+
+const extraRedirects = [
+  {
+    from: '/grunnleggende/om-designsystemet',
+    to: '/no/fundamentals/introduction/about-the-design-system',
+  },
+  {
+    from: '/monstre/representasjon',
+    to: '/no/patterns/representation',
+  },
+  {
+    from: '/monstre/obligatoriske-og-valgfrie-felt',
+    to: '/no/patterns/required-and-optional-fields',
+  },
+  {
+    from: '/monstre/systemvarsler',
+    to: '/no/patterns/systemnotifications',
+  },
+  {
+    from: '/monstre/feilmeldinger',
+    to: '/no/patterns/errors',
+  },
+  {
+    from: '/bloggen/2024/altinn-studio',
+    to: '/no/blog/2024/altinn-studio',
+  },
+  {
+    from: '/bloggen/2024/bachelor-temavelger',
+    to: '/no/blog/2024/bachelor-themeselector',
+  },
+  {
+    from: '/bloggen/2024/fluid-typography',
+    to: '/no/blog/2024/fluid-typography',
+  },
+];
```

Comment:
Suggest we don't have extra redirects. Although it sounds nice, this just opens a can of worms and confusion as some site would redirect and others go to 404 page. This also encourages end users to not update their links and internally this would be a mess over time trying to maintain.

Reply:
This is not used, and simply there if we wish to use it later down the lane. Adding these URLs took 10 minutes, so we are saving some time in the future.

Reply:
Maybe comment it out then? Looks to be used, and I don't want to encourage this behviour 🤔